deltachat-ffi/deltachat.h

```diff
@@ -629,9 +629,8 @@ int             dc_preconfigure_keypair        (dc_context_t* context, const cha
  *   messages from addresses that have no relationship to the configured account.
  *   The last of these messages is represented by DC_CHAT_ID_DEADDROP and you can retrieve details
  *   about it with dc_chatlist_get_msg_id(). Typically, the UI asks the user "Do you want to chat with NAME?"
- *   and offers the options "Yes" (call dc_create_chat_by_msg_id()), "Never" (call dc_block_contact())
- *   or "Not now".
- *   The UI can also offer a "Close" button that calls dc_marknoticed_contact() then.
+ *   and offers the options "Yes", "Never" or "Not now".
```

Comment:
Aren't these options called "Ok", "Not now" and "Block" on Android at least?

Reply:
yes (almost: ¨Ok", "Not now" and "Block contact"); for mailing lists it's "Never" instead of "Block contact" currently to make clear that this will block the mailing list, not the contact (question "Show mailing list with %1$s in Delta Chat?").

On Desktop it's as described in the doc above (yes, not now, never).

Reply:
i'd use the "block" wording also for mailing list.

without knowing that the list was "blocked", it will be harder for the user to understand things are in the "blocked" list.

and i would use this also for the options in the api, i think `StartChat/Block/NotNow` are  more clear answers to a "decision" as `Yes/Never/NotNow`. of course, the ui can tweak the buttons to "Block contact" or maybe "Block mailing list"